app/components/shared/Hotkey.vue

```diff
@@ -59,7 +58,8 @@
   .transition();
 }
 
-.bindings-enter, .bindings-leave-to {
+.bindings-enter,
+.bindings-leave-to {
```

Comment:
not sure why there's a diff here? i get that it's a formatting thing but it also dirties up the commit, i can't tell if there were any actual changes to this component

Reply:
Yes, sorry about the formatting. 
We've added this: https://github.com/stream-labs/streamlabs-obs/pull/1442/files#diff-452d97a23b7ef7b145ddee5b609dd550R2